src/app/utils/institutions.ts

```diff
@@ -0,0 +1,35 @@
+import dynamoClient from "./libs/dynamodb-lib";
+// types
+import { College, CollegeType } from "../types";
+
+const INSTITUTIONS_TABLE_NAME = "institutions";
+
+export async function getInstitutions() {
+  let colleges: College[] = [];
+  const params = {
+    TableName: INSTITUTIONS_TABLE_NAME,
+    FilterExpression: "recordType = :recordType",
+    ExpressionAttributeValues: { ":recordType": "data" },
+    Limit: 10,
```

Comment:
Limit is arbitrary. I assume we get 4 because that hits the scan size limit. It gives us a key to continue scanning once we have the need to display more. For now this is just a tool to get us towards a large view of institutions.